es/index.go

```diff
@@ -422,6 +422,7 @@ type ScrollResult struct {
 	MdbUid     string
 	Id         string
 	ResultType string
+	ScrollId   string
 }
 
 func (index *BaseIndex) Scroll(indexName string, elasticScope elastic.Query) ([]ScrollResult, error) {
```

Comment:
where are all the places this is being used?

Reply:
(1) Line 440: ret = append(ret, ScrollResult{MdbUid: result.MDB_UID, Id: h.Id, ResultType: result.ResultType, ScrollId: searchResult.ScrollId})
(2) Line 484: index.esc.ClearScroll(scrollResult.ScrollId)